### ISSUE TITLE
A11y_code.visualstudio.com_Docs Languages_Java_Keyboard:When keyboard tab focus goes to 'Java Setting started Tutorial' then the color contrast ratio of link is below 3:1.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/vscode-docs#3718]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

 

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.4.11;#Win10;#Visual Studio Code;#CAITestPass_May2020;

 
**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

**Repro Steps:**
1. Launch the application Url :https://code.visualstudio.com
2. Navigate to "Docs" link.
3. From left navigation, select "Languages".
4. Navigate to "Java" link.
5.Open AI4D and select third tab color contrast.
6.Start navigating using keyboard tab key to 'Java setting started Tutorial' link and check color contrast ratio of link is equal or above 3:1,or not.

 
**Actual:**
When keyboard tab focus goes to 'Java Setting started Tutorial' then the color contrast ratio of link is below 3:1.

**Similar issue found in below link in the same screen .**
1)Download Visual Studio Code Java Pack Installer
2)Install the Java Extension Pack

**Expected:**
When keyboard tab focus goes to 'Java Setting started Tutorial' then the color contrast ratio of link should be equal or above 3:1.

**Attachment link for Reference:**
![image](https://user-images.githubusercontent.com/65954864/83003849-834ba780-a02c-11ea-9b65-7797a98f05fd.png)

**Comment by SwatiTiwari-TCS** [comment]
![3718_A11y_code visualstudio com_Docs Languages_Java_Keyboard](https://user-images.githubusercontent.com/65954864/82991574-812d1d00-a01b-11ea-8dbc-39761e1a279b.png)

**Comment by JacksonKearl** [comment]
Thanks for the report!

![image](https://user-images.githubusercontent.com/8586769/87481177-b26da680-c5e3-11ea-8674-e247d8815de6.png)

**Comment by ManishaC1** [comment]
Verified the issue in below environment.
#URL: https://code.visualstudio.com/
Issue is fixed.
Github Tags: #Closed
![Screenshot (440)](https://user-images.githubusercontent.com/105198219/217179199-60f3e059-47f8-422d-88dd-dc90627e6d86.png)
